packages/pds-ember/addon/components/pds/external-link/index.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@glimmer/component'
+
+/**
+ * @class PdsExternalLink
+ */
+
+/**
+ * Controls visibility of the trailing icon.
+ *
+ * @argument hideIcon
+ * @type {boolean}
+ * @default false
+ */
+export default class PdsExternalLink extends Component {
+  get showIcon() {
+    let { hideIcon = false } = this.args
+    return !hideIcon
```

Comment:
Lol the boolean flipping here feels ping-pong-y

Reply:
I've updated to use `unless` instead.